epub34/a11y/index.html

```diff
@@ -520,18 +513,17 @@ <h4>WCAG conformance requirements</h4>
 						while still helpful, can result in a less optimal reading experience.</p>
 
 					<p>Similarly, legal frameworks and policies often cite Level AA conformance as the benchmark for
-						accessibility. The reason is that it provides the greatest range of improvements that EPUB
-						creators can realistically implement. When EPUB creators meet only Level A conformance, they
-						compromise their content for various user groups, resulting in a less optimal reading
-						experience.</p>
+						accessibility as this level provides the greatest range of improvements that are realistic to
+						implement. When only meeting Level A conformance, accessibility is compromised for various user
+						groups, resulting in a less optimal reading experience.</p>
 
 					<div class="note">
 						<p>The <a href="https://www.w3.org/WAI/about/groups/agwg/">W3C Accessibility Guidelines Working
 								Group</a> is currently developing WCAG 3. As this version potentially represents a
 							significant departure from WCAG 2, a future version of this specification will address
-							conformance requirements related to it. EPUB creators are encouraged to adopt WCAG 3 once it
-							is stable and widely recognized, but conformance to the new version is not a requirement of
-							this standard.</p>
+							conformance requirements related to it. Adoption of WCAG 3 is encouraged once it is stable
+							and widely recognized, but conformance to the new version is not a requirement of this
+							standard.</p>
```

Comment:
```suggestion
							conformance requirements related to it. Adoption of WCAG 3 is encouraged, but conformance to the that version is not a requirement of this
							standard.</p>
```

(This goes beyond this PR). I think we should consider EPUB 3 as stable and widely recognized by now...

Reply:
You've flagged a reference to WCAG 3. Did you mean to apply this to a different change or is it just a misread? (WCAG 3 is far from being ready for use.)

Reply:
Oh sht... I completely misread, and I mentally did an WCAG 3 -> EPUB 3 change. 

Sorry, I am tired....

Reply:
> Sorry, I am tired....

Lol, I feel your pain! Editing while fighting off a cold isn't much fun...